app/code/Magento/Eav/Model/Entity/Attribute/AbstractAttribute.php

```diff
@@ -788,6 +796,35 @@ public function getFlatColumns()
         return $this->_getFlatColumnsDdlDefinition();
     }
 
+    /**
+     * Processing object after load data
+     *
+     * @return $this
+     */
+    public function _afterLoad()
+    {
+        parent::_afterLoad();
+
+        $this->_addFrontendLabels();
+
+        return $this;
+    }
+
+    /**
+     * Add Frontend Labels information
+     *
+     * @throws LocalizedException
+     */
+    public function _addFrontendLabels()
```

Comment:
Please, don't use underscores in method names and make this method private

Reply:
I'm not sure making the method private is a good idea based on:
- Accessing this method might be useful for loading the labels in case they weren't (i.e: loading a collection of attributes)
- The method `_afterLoad` is public, and I don't think making this method private would be a good idea... it might be a problem in case someone wants to override the `afterLoad` method

What do you think?

Reply:
Currently, this method used only in this class and should be private, if you have other cases when you need frontend labels you should extract this method to separate class and use in all client code, but for now, I think it will be redundant and changing this method to private will be enough.